### ISSUE TITLE
habbo-web-l10n says "Unauthorized"

### ISSUE THREAD

**Issue by T0niiiiii** [sant0ro/chocolatey#342]
https://onnelahotelli.com/habbo-web-l10n/fi.json 

so what now? 

![sieppaa](https://user-images.githubusercontent.com/30376926/32015247-2bb8a65e-b9c9-11e7-9e35-28bf7180dfab.PNG)

![error](https://user-images.githubusercontent.com/30376926/32015169-f2d47e30-b9c8-11e7-95eb-b7b13cc7e6bd.PNG)

**Comment by sant0ro** [comment]
Yes but this folder need to be protected. This is not an error. The actual error is when you access this file happens "https://onnelahotelli.com/habbo-web-l10n/fi.json" Error 500.

Show me the contents of your `lumen.log` file. Probably this `fi.json` file doesn't exists. OR you misconfigured it.

**Comment by sant0ro** [comment]
Can you send me the content of the `fi.json` file and the content of `storage/logs/lumen.log`?

**Comment by T0niiiiii** [comment]
[data.zip](https://github.com/sant0ro/chocolatey/files/1415928/data.zip)

**Comment by sant0ro** [comment]
Can you clear the lumen.log file and resend it with a single attempt of opening fi.json in the browser?

**Comment by T0niiiiii** [comment]
I clear the log, but now it's only empty

**Comment by sant0ro** [comment]
So I really don't know what is the problem. If gives Error 500 and doesn't store the error. Can you check directly apache's errors?

Here is what the Language System does: https://github.com/sant0ro/chocolatey/blob/master/app/Http/Controllers/LanguageController.php

By a weird reason it seems that any language it's giving error 500. So check please Apache's `error_log` file.

And give me the content. (I recommend delete the file first, try to access the `fi.json` and show the results of the error log file here)
